apps/api/src/app/inbox/dtos/button-type-request.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { IsDefined, IsEnum } from 'class-validator';
+import { ButtonTypeEnum } from '@novu/shared';
+
+export class ButtonTypeRequestDto {
```

Comment:
How about `actionType` to keep it more generic. A custom notification item, might not always have a button.

Reply:
This also aligns the naming with the response type of primary and secondary actions.